packages/xo-web/src/xo-app/settings/audit/index.js

```diff
@@ -371,6 +378,19 @@ export default decorate([
             {_('auditCheckIntegrity')}
           </ActionButton>
         </div>
+        {state.isUserActionsRecordInactive && (
+          <Link
+            className='text-warning'
+            to={{
+              pathname: '/settings/plugins',
+              query: {
+                s: 'id%3A%2F^audit%24%2F',
```

Comment:
No need to encode it when you pass it as a query object.

Reply:
@julien-f

Reply:
```suggestion
                s: 'id:/^audit$/',
```

Reply:
My mistake, please try @pdonias' suggestion.

Reply:
Ah ok you want a regex expression because i don't understand the encoding.

Reply:
```
> node -p "decodeURIComponent('id%3A%2F^audit%24%2F')"
id:/^audit$/
```

Reply:
Ah ok because usally i use `encodeURIComponent`